ApiBundle/Controller/DeletedController.php

```diff
@@ -24,15 +30,12 @@ class DeletedController extends BaseController
      *
      * @return Response
      */
-    public function listAction()
+    public function listAction(Request $request)
     {
-        $siteId = $this->get('open_orchestra_backoffice.context_manager')->getCurrentSiteId();
-        /** @var array $nodes */
-        $nodes = $this->get('open_orchestra_model.repository.node')->findLastVersionByDeletedAndSiteId($siteId);
-        $contents = $this->get('open_orchestra_model.repository.content')->findAllDeleted();
-
-        $deleted = array_merge($nodes, $contents);
+        $mapping = $this->get('open_orchestra_base.annotation_search_reader')->extractMapping($this->getParameter('open_orchestra_model.document.trash_item.class'));
```

Comment:
This way of getting the mapping should be done all the time, is there a ticket for that ?